app/models/household.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# A collection of family members / possible users
+class Household < ApplicationRecord
+    # Active Record implicitly creates the primary key as 'id.' 
+
+    # Determines table relationships. We can alias household_address -> address
+    has_one :address, class_name: "HouseholdAddress", :dependent => :destroy
```

Comment:
Either make this `household_address` or rename the model to `Address`. That way the framework can infer the relationship and we don't need class_name.